Entities/Items/Projectiles/Arrow.as

```diff
@@ -545,12 +545,17 @@ void FireUp(CBlob@ this)
 	if (map !is null)
 	{
 		// burninate
+		/*
 		if (!isTeamStructureNear(this))
 		{
 			map.server_setFireWorldspace(burnpos, true);
 			map.server_setFireWorldspace(this.get_Vec2f("fire pos") + head * 0.4f, true);
 			map.server_setFireWorldspace(this.getPosition() , true); //burn where i am as well
 		}
+		*/
```

Comment:
Don't worry about commenting stuff like this out - it's in the git history if needed